frontend/src/store/index.js

```diff
@@ -356,13 +369,19 @@ const getters = {
           return semver.rcompare(a.version, b.version)
         })
 
-        return map(versions, ({ version, expirationDate }) => {
+        return map(versions, ({ version, expirationDate, classification }) => {
           const vendorName = vendorNameFromImageName(machineImage.name)
           const name = machineImage.name
+
           return {
             key: name + '/' + version,
             name,
             version,
+            classification,
+            isPreview: classification === 'preview',
+            isSupported: classification === 'supported',
+            isDeprecated: classification === 'deprecated',
+            isExpired: expirationDate && moment().isAfter(expirationDate),
```

Comment:
this won't be reactive, right? But nothing that can be changed easily I guess and it's a corner case

Reply:
correct - but this prop is currently only used to filter out expired versions, which a user should not use anyways, especially not if they are about to expire. I think we can live with the limitation that we sometimes show expired versions. If we want to have this reactive we could move the code to a utility function and call it instead of pre-calculating the value. WDYT?

Reply:
yes agree, we leave it as is